imap_processing/cli.py

```diff
@@ -580,9 +579,7 @@ def do_processing(self, dependencies):
                 )
             # read CDF file
             processed_data = PacketParser(dependencies[0], self.version).data
-            cdf_file_path = write_cdf(processed_data)
-            print(f"processed file path: {cdf_file_path}")
-            return [cdf_file_path]
+            return [processed_data]
```

Comment:
```suggestion
            return [dataset]
```

There are only a few instruments that are using a different variable name other than `dataset` here. Should we change the them so all instruments are consistent?

Reply:
Yes, great suggestion. I just updated all of the returns to be either `[dataset]` or `datasets` depending on whether one or multiple were being returned. Maybe we should update that later as well to force us to always return a list from the calling procedure... But that will lead to more discussion so lets save it for a follow-up PR.